catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/TransactionRequestConverter.java

```diff
@@ -332,14 +350,10 @@ private Serializable getRecordPropertyValue(HierarchicalStreamReader reader, Str
             Serializable newValue;
             if (reader.hasMoreChildren()) {
```

Comment:
no. this element is an `anyType` but it must be constrained within a `<Value>`.  At least how I'm reading the schema the `<Value>` element can repeat but the content of that element cannot.